src/wallet/scriptpubkeyman.cpp

```diff
@@ -421,47 +418,47 @@ bool CWallet::AddKeyPubKeyWithDB(WalletBatch& batch, const CKey& secret, const C
                                                  secret.GetPrivKey(),
                                                  mapKeyMetadata[pubkey.GetID()]);
     }
-    UnsetWalletFlagWithDB(batch, WALLET_FLAG_BLANK_WALLET);
+    m_storage.UnsetWalletFlagWithDB(batch, WALLET_FLAG_BLANK_WALLET);
     return true;
 }
 
-bool CWallet::LoadCScript(const CScript& redeemScript)
+bool LegacyScriptPubKeyMan::LoadCScript(const CScript& redeemScript)
 {
     /* A sanity check was added in pull #3843 to avoid adding redeemScripts
      * that never can be redeemed. However, old wallets may still contain
      * these. Do not add them to the wallet and warn. */
     if (redeemScript.size() > MAX_SCRIPT_ELEMENT_SIZE)
     {
         std::string strAddr = EncodeDestination(ScriptHash(redeemScript));
-        WalletLogPrintf("%s: Warning: This wallet contains a redeemScript of size %i which exceeds maximum size %i thus can never be redeemed. Do not use address %s.\n", __func__, redeemScript.size(), MAX_SCRIPT_ELEMENT_SIZE, strAddr);
+        m_wallet.WalletLogPrintf("%s: Warning: This wallet contains a redeemScript of size %i which exceeds maximum size %i thus can never be redeemed. Do not use address %s.\n", __func__, redeemScript.size(), MAX_SCRIPT_ELEMENT_SIZE, strAddr);
```

Comment:
In commit "Refactor: Split up CWallet and LegacyScriptPubKeyMan and classes" (194227b1173b0dd9a51d24e245ddaa601d36a109)

Would be better not to reference `m_wallet` here. `m_wallet` is a temporary alias so any references added now will have to be removed later, causing unnecessary churn. Would suggest reverting the commit back to your previous af8780a65a530c48a0abc7f3ef6915f33d30943f push, and just adding the linter fixes in 626eed831918ac4f492b94ad0131f65d67fd1785

Reply:
Reverted and squashed linter changes.